frontend/src/screens/learner.tsx

```diff
@@ -172,7 +172,6 @@ export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }
             <Swiper
                 slidesPerView={3}
                 simulateTouch={true}
-                freeMode={true}
```

Comment:
would you be able to explain why you removed this?

Reply:
The slides were overlapping, so I wanted to see if any of the properties in the swiper component are causing the issue, so  I removed everything and kept checking if it would solve the issue. I found that the overlapping issue would be solved if the width of the cards were reduced, freeMode can be added back. I overlooked that.